buildout.cfg

```diff
@@ -9,11 +9,14 @@ parts =
 
 [versions]
 Products.PythonScripts =
+AccessControl =
+RestrictedPython =
```

Comment:
Shouldn't we be relying on the now-released versions published via https://raw.githubusercontent.com/zopefoundation/Zope/master/versions.cfg (or even better, https://zopefoundation.github.io/Zope/releases/master)?